base_frame: world
tool_frame: tip
max_penalty: 5.0
max_distance: 0.5
min_distance: 0.0
links: ["ur5_tool0","tip","ur5_wrist_1_link"]
computation_step: 0.04

local_bias: 0.5
tube_radius: 0.1

